src/public/Interfaces.ts

```diff
@@ -1,54 +1,150 @@
+/**
+ * The Azure Function that is exported (via module.exports or exports). This is the function that will 
```

Comment:
Can we use same description as [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node)?